projects/public-holdings-items/src/app/app-initializer.service.ts

```diff
@@ -49,7 +49,7 @@ export class AppInitializerService {
   private _initiliazeObservable(): void {
     // Set current language interface
     this._userService.loaded$.subscribe(() => {
-      this._translateService.setLanguage(this._appSettingsService.currentLanguage);
+      this._translateService.use(this._appSettingsService.currentLanguage);
```

Comment:
I don't know why. Is that means that `setLanguage` is useless? `setLanguage` do the `use` and many more.

Reply:
`setLanguage` is the rero translate service and `use` is for ngx-translate